README.md

```diff
@@ -11,67 +11,13 @@ launching the framework <i>scheduler</i>.
 
 The framework can be run by building the code, the docker images, transferring the code to the Mesos cluster and launching the framework <i>scheduler</i>.
 
-> $ gradlew build
+## How to build
 
-Optionally run <a href="https://github.com/dougborg/gdub">gdub</a> which runs the gradle wrapper from any subdirectory.
-
-## How to build on Mac
-
-Actions to perform to start in Mac:
-
- 1. In terminal go to your project directory:
-	> $ cd project_directory
-
- 2. Start Vagrant:
-	> $ vagrant up
-	
- 3. When completed the installation/startup then SSH:
-	> $ vagrant ssh
-	
- 4. Get to project directory in VM:
-	> $ sudo -i
-	
-	> $ cd /vagrant
-
- 5. Start the build with Gradle:
-	> $ ./gradlew build
-
-If you get the following failure 
->What went wrong:
-Execution failed for task ':elasticsearch-cloud-mesos:buildDockerImage'. 
-\> org.newsclub.net.unix.AFUNIXSocketException: No such file or directory (socket: /run/docker.sock)
-
-then run the following command to make sure that docker is running:
-	> $ systemctl restart docker
-
-and finally run gradlew build again:
-	> $ ./gradlew build
-
-## How to build with vagrant
-
-> Known to work on Vagrant 1.7.2 and VirtualBox 4.3.26
-
-> You can save yourself some time by first checking that the docker service is running with
-> `systemctl status docker`. If it is not, you can start it using `sudo systemctl start docker`
-
-> Currently you need to sudo the build command or the Docker part will fail. This will be fixed shortly.
-
-The following will build everything.
-
-````
-$ vagrant up
-$ vagrant ssh
-$ cd /vagrant
-$ sudo ./gradlew build
-````
-
-You can also build sub-components by remembering to use a relative path to the gradle wrapper. If you don't want to deal with relative paths you can run <a href="https://github.com/dougborg/gdub">gdub</a> which runs the gradle wrapper from any subdirectory.
-
-````
-$ cd /vagrant
-$ cd commons
-$ sudo ../gradlew build
-````
+```
+$ docker run --rm -v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
In fact it does. Because docker runs in the context of boot2docker (virtualbox) image, and it is there (inside the VM).